### ISSUE TITLE
Serbian Cyrillic wrong font

### ISSUE THREAD

**Issue by medolino2009** [immich-app/immich#11216]
### The bug

Hi guys, i am translator of Serbian latin and Cyrillic language, Immich renders whole Cyrillic with "wrong" font...very hard to read when you have it in Serbian Cyrillic... 
I tried to change language to Russian and that language renders with readable font for example... I searched google fonts and "Alumni Sans" font renders nice readble Serbian Cyrillic. I hope you can change this. 

Mobile app don't support Serbian Cyrillic yet (i can choose Serbian Cyrillic but language changes to English), if this also could be fixed, would be splendid

btw. Thanx a lot for amazing software, i've already got license for it. Keep up great job.!!!

Thanx in advance.

Best regards

Miki

### The OS that Immich Server is running on

Unraid docker server

### Version of Immich Server

v.1.109.2

### Version of Immich Mobile App

v.1.109.0

### Platform with the issue

- [ ] Server
- [x] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
It is translation trouble
```

### Your .env content

```Shell
It is translation trouble
```

### Reproduction steps

```bash
1.Change to Serbian Cyrillic in web
2.Language changes to Serbian Cyrillic
3.Font is dificult to read
...
```

**Comment by jrasm91** [comment]
Are you saying that the current font does not look/work very well in that language? Can you show screenshots of it and pictures of it with the new font?

**Comment by medolino2009** [comment]
> Are you saying that the current font does not look/work very well in that language? Can you show screenshots of it and pictures of it with the new font?

Yes, that is exactly what i was saying

This is wrong font (dificult to read because of some letters):
![Immich cyrillic wrong font](https://github.com/user-attachments/assets/69f2b66a-87ff-431f-87f6-1d20ad4a778c)

This is font that works wel in Immich:
![Immich Alumni Sans font](https://github.com/user-attachments/assets/e382e1fd-a27d-411f-bb64-245bdd7714d3)

i think Alumni Sans Regular 400 looks best.

Thanx in advance.

Best regards

Miki

**Comment by VelocityRa** [comment]
@medolino2009 Does Serbian Latin render incorrectly too?

**Comment by meesfrensel** [comment]
Immich recently changed to a different font. Does that work better for cyrililc?

**Comment by VelocityRa** [comment]
> Immich recently changed to a different font. Does that work better for cyrililc?

Yes! At least on my end on one of the devices where I had the issue (Samsung S23), Greek now works well!

**Comment by meesfrensel** [comment]
Thanks for confirming
